java/clients/src/main/java/sleeper/clients/admin/deploy/UpdateExistingInstance.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2022-2023 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sleeper.clients.admin.deploy;
+
+import com.amazonaws.services.ecs.AmazonECS;
+import com.amazonaws.services.ecs.AmazonECSClientBuilder;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import software.amazon.awssdk.services.s3.S3Client;
+
+import sleeper.configuration.properties.InstanceProperties;
+import sleeper.configuration.properties.local.LoadLocalProperties;
+import sleeper.configuration.properties.local.SaveLocalProperties;
+import sleeper.core.SleeperVersion;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+public class UpdateExistingInstance {
```

Comment:
It might be more consistent to use the word "deploy" rather than "update", to match the CDK command we're using. The same applies to the name of the bash script.